merlin/models/tf/blocks/dlrm.py

```diff
@@ -34,52 +34,55 @@ def DLRMBlock(
     *,
     embedding_dim: int = None,
     embedding_options: EmbeddingOptions = None,
+    embeddings: Optional[Block] = None,
```

Comment:
Let's not move the order of arguments if possible - it's a breaking change in the unlikely case where people provide all of the arguments in order as opposed to using named arguments.

Reply:
@nv-alaiacano thanks for your comment. Make sense, I talked Oliver, I think we are safe changing the position of the arg thanks to `*` arg there. It makes everything forced to be a keyword arg, rather than a positional arg.